test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -635,7 +635,7 @@ public void flush(String keyspace)
     {
         ColumnFamilyStore store = getCurrentColumnFamilyStore(keyspace);
         if (store != null)
-            store.forceBlockingFlush();
+            store.forceBlockingFlush(ColumnFamilyStore.FlushReason.UNIT_TESTS);
```

Comment:
I guess we're going to have to change a ton of tests either way, so what do you think about adding a couple utility flush methods to `CQLTester` that hide the details of the flush reason?

ex.

```
List<Future<?>> flush(Keyspace keyspace)
{
    return keyspace.flush(ColumnFamilyStore.FlushReason.UNIT_TESTS);
}
```

Reply:
...probably also one for `forceBlockingFlush()`